java/vector/src/main/java/org/apache/arrow/vector/complex/AbstractStructVector.java

```diff
@@ -328,4 +328,40 @@ public int getBufferSize() {
     }
     return actualBufSize;
   }
+
+  @Override
+  public String toString() {
+    if (getValueCount() == 0) {
+      return "[]";
+    }
+
+    final int window = 10;
+    boolean skipComma = false;
+
+    StringBuilder sb = new StringBuilder();
+    sb.append('[');
```

Comment:
It seems everything is in the same line. This is intended?

Reply:
yes, since it could be truncated, the length won't be too long, I made it in a line which is consistent with java.util.Collection#toString.